### ISSUE TITLE
Add logging support in MBraceClient

### ISSUE THREAD

**Issue by krontogiannis** [mbraceproject/MBrace.Core#120]
[For MBrace.Runtime.Core 0.9.14]

It would be useful to have methods for fetching system logs in `MBraceClient`:
```fsharp
GetSystemLogs : unit -> LogEntry
ShowSystemLogs : unit -> unit
```
and fetching cloudlogs per process/CloudTask
```fsharp
GetCloudLogs : unit -> LogEntry
ShowCloudLogs : unit -> unit
```
Either add a Get method on ISystemLogger i guess, or just add an abstract method on `MBraceClient`

**Comment by eiriktsarpalis** [comment]
Fixed in f6f18236be9fe780a9a54a3539b07326174b10a2

**Comment by eiriktsarpalis** [comment]
I have not included facility to fetch system logs for individual nodes, since this seems a bit too implementation specific.
